x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/endpoint_action_response_codes.ts

```diff
@@ -276,28 +276,28 @@ const CODES = Object.freeze({
 
   'ra_scan_error_scan-invalid-input': i18n.translate(
     'xpack.securitySolution.endpointActionResponseCodes.scan.invalidInput',
-    { defaultMessage: 'Scan failed. Invalid absolute file path provided.' }
+    { defaultMessage: 'Invalid absolute file path provided' }
```

Comment:
While doing manual testing I got an error from the endpoint with a code of `ra_scan_error_invalid-input`, but here (line 277) its defined as `*-invalid-input` (dash instead of underscore. can you correct this?

Reply:
@matthewscherer I'm now noticing that the invalid input and queue quota error codes have an extra `_scan` after `_error` in the code, compared to the not found error code. Is that on purpose? I used the example codes that you provided [here](https://github.com/elastic/security-team/issues/8993#issuecomment-2141411217).

Reply:
That is not on purpose. Let me take a look at it.

Reply:
I  fixed the `ra_scan_error_queue-quota` but the `ra_scan_error_invalid-input` seems right. Looking at the rest of the error codes, they look like [this](https://github.com/elastic/endpoint-dev/blob/main/Plugins/ResponseActions/Lib/ResponseActionsCommon.cpp#L187C1-L254C6). I suspect `*-invalid-input` should be `*_invalid-input`

PR for the fix is here. https://github.com/elastic/endpoint-dev/pull/14549

Reply:
Thanks @matthewscherer I fixed the invalid input code in 99e13666cb030e03427e48b3bf935b09b293207d